packages/eslint-plugin-ecmascript-compat/lib/rule.js

```diff
@@ -11,7 +11,20 @@ const delegateeConfigs = compatibility
 module.exports = {
   meta: {
     type: 'problem',
-    schema: [], // no options
+    schema: [
+      {
+        "type": "object",
+        "properties": {
+          "pollyfils": {
```

Comment:
The spellings are "polyfills" and "polyfill" for a single one.

It's from [Polyfilla](https://en.wikipedia.org/wiki/Spackling_paste#Polyfilla) by the way :); we are filling holes in the browser.